torchbenchmark/models/pyhpc_turbulent_kinetic_energy/__init__.py

```diff
@@ -129,6 +129,7 @@ class Model(BenchmarkModel):
     # Source: https://github.com/dionhaefner/pyhpc-benchmarks/blob/650ecc650e394df829944ffcf09e9d646ec69691/run.py#L25
     # Pick data-point when i = 20, size = 1048576
     DEFAULT_EVAL_BSIZE = 1048576
+    DEFAULT_TRAIN_BSIZE = 1048576
```

Comment:
We do not need to add train bsize because the `train()` function is not implemented or supported by this model.